.github/workflows/on_workflow_dispatch.yaml

```
@@ -0,0 +1,31 @@
+name: Manually Run Rock Integration
+
+on:
+  workflow_dispatch:
+    inputs:
+      rock-dir:
+        description: "Path to the rock directory (e.g. rocks/my-rock)"
+        required: true
+        type: string
+      target-branch:
+        description: "Branch to integrate into"
+        required: true
+        type: string
+      dry-run:
+        description: "Dry run?"
+        required: false
+        type: boolean
+        default: true
+      full-image-tag:
+        description: "Full image reference from Docker registry"
```

nit: it doesn't have to be docker right?
```suggestion
        description: "Full image reference from Image registry"
```